### ISSUE TITLE
Read back P/T information from PassiveTracers to julia

### ISSUE THREAD

**Issue by boriskaus** [JuliaGeodynamics/LaMEM.jl#34]
We can now activate passive tracers in the Julia interface, which will generate paraview output. Yet, we do not have a simple way yet to read certain tracers (for example all tracers in the oceanic crust) back to Julia and plot a P-T path etc.

**Comment by boriskaus** [comment]
In fact this turns out to be not true. 
We can already read back passive tracers with the function `Read_LaMEM_timestep`.
Here a worked-out example:

```Julia
julia> using LaMEM, GeophysicalModelGenerator
julia> # 2D model setup
       model = Model(Grid( x = [-1000.,1000.],  z = [-1000,0] , nel = (256,128)),
                           BoundaryConditions( temp_bot = 1350.0, temp_top = 0.0),
                           Output(out_temperature=1, out_j2_dev_stress=1, out_pressure=1),
                           PassiveTracers(Passive_Tracer=1),                            
                           Solver(DirectSolver="mumps"),
                           ModelSetup( nmark_lim      = [27, 64]),
                           Time(nstep_max=100, time_end=10, nstep_out=5) )  
LaMEM Model setup
|
|-- Scaling             :  GeoParams.Units.GeoUnits{GEO}
|-- Grid                :  nel=(256, 1, 128); xϵ(-1000.0, 1000.0), yϵ(-3.90625, 3.90625), zϵ(-1000.0, 0.0) 
|-- Time                :  nstep_max=100; nstep_out=5; time_end=10.0; dt=0.05
|-- Boundary conditions :  noslip=[0, 0, 0, 0, 0, 0]
|-- Solution parameters :  eta_min=1.0e18; eta_max=1.0e25; eta_ref=1.0e20; act_temp_diff=0
|-- Solver options      :  direct solver; mumps; penalty term=10000.0
|-- Model setup options :  Type=files; 
|-- Output options      :  filename=output; pvd=1; avd=0; surf=0
|-- Passive Tracers     :  Type=Always; Res=[100,1,100], Box=[-1000.0:1000.0,-3.90625:3.90625,-1000.0:0.0]
|-- Materials           :  0 phases; 

Tair, Tmantle      = 0.0, 1350.0;
model.Grid.Temp   .= Tmantle;      # set mantle temperature (without adiabat at first)
model.Grid.Phases .= 0;   

# Add temperature & layers in lithosphere
AddBox!(model;  xlim    = (model.Grid.coord_x...,),  zlim    = (model.Grid.coord_z...,),
                phase   = LithosphericPhases(Layers=[30 80], Phases=[0 1 2] ),
                T       = HalfspaceCoolingTemp(Tsurface=0, Tmantle=Tmantle, Age=350))

# Add "slab" 
AddBox!(model;  xlim    = (-150,150), zlim    = (-400,-350), 
                Origin  = (0,0,-400), DipAngle=20,
                phase   = ConstantPhase(3))

# Add phase transitions ---
PT0 = PhaseTransition(ID=0, Type="Constant", Parameter_transition="T",  ConstantValue=1200, PhaseBelow = [1], PhaseAbove=[2], PhaseDirection="BothWays")
PT1 = PhaseTransition(ID=1, Type="Constant", Parameter_transition="Depth", ConstantValue=-660, PhaseBelow = [4], PhaseAbove=[2], PhaseDirection="BothWays")
model.Materials.PhaseTransitions=[PT0, PT1]
# -------------------------                 
                
# add material properties
ρ0 = 3200.0;
Δρ = 00.0;
η0 = 1e21;
γ  = 5e-3;
T0 = 1350;
α   = 3e-5;

# Set material properties
Mantle = Phase(ID=2,Name="Mantle",eta_fk=1e20,gamma_fk=γ,TRef_fk=T0,rho=ρ0, alpha=α);
Craton = copy_phase(Mantle, ID=1, Name="Craton", rho=ρ0+Δρ)
Crust  = copy_phase(Mantle, ID=0, Name="Crust", rho=ρ0+Δρ)
Slab   = Phase(ID=3,Name="Slab",eta=1e22,rho=ρ0+Δρ);
LMantle= Phase(ID=4,Name="LowMantle",eta_fk=5e21,gamma_fk=γ,TRef_fk=T0,rho=ρ0, alpha=α);

# add materials to model
rm_phase!(model)
add_phase!(model, Mantle, Craton, Slab, Crust, LMantle)

# run model
run_lamem(model, 1)
```

**Comment by boriskaus** [comment]
closed with PR #35
